articles/api-auth/tutorials/adoption/delegation.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Delegation and the OIDC-conformant pipeline
+---
+
+<%= include('./_about.md') %>
+
+# Delegation and the OIDC-conformant pipeline
+
+[Delegation](/api/authentication#delegation) is used for many operations, depending on your particular use case:
+
+* Exchanging an ID token issued to one client for a new one issued to a different client
```

Comment:
We should explain how this particular use case should no longer be necessary anyway with the new pipeline. Something like:

> Typically this was done in the old pipeline because the API itself was represented as a client in Auth0. Therefore a frontend client would need to use the delegation endpoint to obtain another `id_token` that had the audience of the API in order to call the API. This became very chatty, requiring lots of calls to get the right `id_token`s. This entire use case has been properly solved with the new pipeline since APIs are now first class entities and `access_token`s can be issued to any number of clients for a single API and will only have that API's identifier as the audience.